Launcher/config.json

```diff
@@ -43,30 +35,23 @@
   "data-channel-provider": "DataChannelProvider",
   "object-store": "QuantConnect.Lean.Engine.Storage.LocalObjectStore",
   "data-aggregator": "QuantConnect.Lean.Engine.DataFeeds.AggregationManager",
-
   // limits on number of symbols to allow
   "symbol-minute-limit": 10000,
   "symbol-second-limit": 10000,
   "symbol-tick-limit": 10000,
-
   // limits the amount of data points per chart series. Applies only for backtesting
   "maximum-data-points-per-chart-series": 4000,
-
   // if one uses true in following token, market hours will remain open all hours and all days.
   // if one uses false will make lean operate only during regular market hours.
   "force-exchange-always-open": false,
-
   // save list of transactions to the specified csv file
   "transaction-log": "",
-
   // To get your api access token go to quantconnect.com/account
   "job-user-id": "0",
   "api-access-token": "",
-
   // live data configuration
   "live-data-url": "ws://www.quantconnect.com/api/v2/live/data/",
   "live-data-port": 8020,
-
```

Comment:
Please let's revert these whitespace changes, leaving only the new zerodha config entries

Reply:
ESLint Issues :) will fix it.